service/accounts/openstack_manager.py

```diff
@@ -441,30 +441,33 @@ def shared_images_for(self, image_id):
                     for member in shared_with]
         return projects
 
-    def share_image_with_project(self, glance_image, project_name):
```

Comment:
Not part of this PR, but I noticed that lines 430 & 431 contain tabs for indentation.